### DISCUSSION TITLE
About Installing Dotfiles

### DISCUSSION THREAD

**Discussion by NoobU-git** [end-4/dots-hyprland#686] [Q&A] [answered]
Can I install this without use any DE? I read on wiki it says EndeavorOS with gnome was recommended. But I need to know, is it works too without any DE installed. I just want to use the hyprland WM with this dotfiles. I want to use this on Archlinux

**Comment by clsty** [comment, marked as answer]
Yes, it works without Gnome. I'm using Arch Linux without Gnome.

**Reply by NoobU-git** [reply]
Thanks, I just installed and it works perfectly  ✨️

**Comment by end-4** [comment]
I just made that recommendation since I do not have the ability to support raw Arch users
Feel free to use whatever Arch of your choice

**Reply by NoobU-git** [reply]
Thanks bro. Already installed and works perfectly

**Reply by clsty** [reply]
Welp, just a reminder, better avoid Manjaro because it's not using the official repos of Arch Linux and sometimes cause issues when user try to install packages from AUR which is for using with official repos of Arch.

Most other Arch based distros are fine though, e.g. EndeavourOS.
